webpack/assets/javascripts/react_app/components/PermissionDenied/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
This file should be renamed to `components/PermissionDenied/PermissionDenied.js` by the foreman conventions.

For easy access, you can have an `index.js`:
```js
import PermissionDenied from './PermissionDenied';

export default PermissionDenied;
```